src/dataflow-types/src/types/sinks.proto

```diff
@@ -38,6 +39,11 @@ message ProtoKafkaSinkConsistencyConnector {
     int32 schema_id = 2;
 }
 
+message ProtoSinkAsOf {
+    mz_persist.gen.persist.ProtoU64Antichain frontier = 1;
```

Comment:
`Antichain` is just a `Vec<u64>`! Well, it technically could have more complicated timestamps, but all timestamps used right now are `u64`.

Reply:
I wonder for how long these u64's remain meaningful...

I think we should definitely move it, to mz-timely-util or mz-repr or something?

Reply:
This actually predates the current Protobuf initiative, and would cause `persist` to depend on `repr` (or whatever the eventual home for this type is), so we might want to chat with @danhhz before.

Reply:
Feel free to fork it, but yes, please leave persist protos alone :). They're used only internally in persist, no other service sees them, so there's no particular reason they need to be shared with the rest of the codebase

Also, heads up that we're going to change this one at some point. It's from old persist, which required that timestamps be u64s. New persist allows them to be anything that implements Codec64, so this will become something like an antichain of `[u8; 8]`